src/ontology/ro-edit.owl

```diff
@@ -42,6 +42,9 @@ Annotation(foaf:homepage " https://github.com/oborel/obo-relations/"^^xsd:anyURI
 Declaration(Class(obo:BFO_0000019))
 Declaration(Class(obo:BFO_0000031))
 Declaration(Class(obo:CHEBI_50906))
+Declaration(Class(obo:CL_0000000))
+Declaration(Class(obo:CL_0000101))
+Declaration(Class(obo:CL_0000540))
```

Comment:
are you referring to these terms? Could you move them to `src/ontology/imports/cl_terms.txt`, please?

Reply:
@pgaudet ping

Reply:
Thanks! I tried to fix this (it was already in the CL import file, so that hasn't changed)

Reply:
Is this now OK to merge?

Reply:
yes